manifests/init.pp

```diff
@@ -302,7 +302,10 @@
 # @param rdb_save_incremental_fsync
 #   When redis saves RDB file, if the following option is enabled
 #   the file will be fsync-ed every 32 MB of data generated.
-
+# @param dnf_module_stream
+#   Manage the DNF module and set the version. This only makes sense on distributions
+#   that use DNF package manager, such as EL8 or Fedora. It also requires Puppet 5.5.20+ or
+#   Puppet 6.15.0+ since they ship the dnfmodule provider.
```

Comment:
Given the minimum version is specified in `metadata.json`, I'd leave it out here
```suggestion
#   that use DNF package manager, such as EL8 or Fedora.
```